packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -225,8 +225,9 @@ function BudgetCell({
   const format = useFormat();
   const { showUndoNotification } = useUndo();
   const [budgetType = 'rollover'] = useSyncedPref('budgetType');
```

Comment:
The syncedpref value needs to be changes to `envelope`/`tracking` so that we don't need to translate everywhere this syncedpref is used